server/apps/archive/archive.py

```diff
@@ -362,17 +367,15 @@ def _duplicate_item(self, original_doc):
         del new_doc[config.ID_FIELD]
         del new_doc['guid']
 
+        new_doc[ITEM_OPERATION] = ITEM_DUPLICATE
         item_model = get_model(ItemModel)
 
         on_duplicate_item(new_doc)
-        item_model.create([new_doc])
-        self._duplicate_versions(original_doc['guid'], new_doc)
-
+        resolve_document_version(new_doc, 'archive', 'PATCH', new_doc)
```

Comment:
Any specific reason for moving this put of if condition? 
We need not to increment the version when the article state is 'Submitted'. 
With this change it looks like for each copy and duplicate operation we increment the version which means there won't be any change in the last two versions.